modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,155 @@
+== Breaking changes
+
+* With this update, the prior version of the `tkn` command is not compatible with {pipelines-title} 1.10.
+* This update removes support for `Cluster` and `CloudEvent` pipeline resources from Tekton CLI. You cannot create pipeline resources by using the `tkn pipelineresource create` command. Also, the pipeline resources are not supported anymore in the `start` command of a task, cluster task, and pipeline.
+* This update removes `tekton` as a provenance format from Tekton Chains.
+
+
+[id="deprecated-features-1-10_{context}"]
+== Deprecated and removed features
+
+* In {pipelines-title} 1.10, the `ClusterTask` commands are now deprecated and are planned to be removed in a future release. The `tkn task create` command is also deprecated with this update. 
+* In {pipelines-title} 1.10, the flags `-i` and `-o` that were used with the `tkn task start` command are now deprecated because the v1 API does not support pipeline resources.
+* In {pipelines-title} 1.10, the flag `-r` that was used with the `tkn pipeline start` command is deprecated because the v1 API does not support pipeline resources.
+* The {pipelines-title} 1.10 update sets the `openshiftDefaultEmbeddedStatus` parameter to `both` with `full` and `minimal` embedded status. Also, the flag to change the default embedded status will be removed and the pipeline default embedded status will be moved to minimal in a future release.
+
+[id="known-issues-1-10_{context}"]
+== Known issues
+
+* This update includes the following backward incompatible changes:
+** Removal of cluster `PipelineResources`
+** Removal of cloud event `PipelineResources` 
+* If the pipelines metrics feature does not work after a cluster upgrade, run the following command as a workaround:
++
+[source,terminal]
+----
+$ oc get tektoninstallersets.operator.tekton.dev | awk '/pipeline-main-static/ {print $1}' | xargs oc delete tektoninstallersets
+----
+* With this update, usage of external databases, such as the Crunchy PostgreSQL is not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}. Instead, use the default {tekton-hub} database. 
+
+[id="fixed-issues-1-10_{context}"]
+== Fixed issues
+
+* Before this update, the `opc pac` command generated a runtime error instead of showing any help. This update fixes the `opc pac` command to show the help message.
+* Before this update, running the `tkn pac create repo` command needed the webhook details for creating a repository. With this update, the `tkn-pac create repo` command does not configure a webhook when your GitHub application is installed.
+* Before this update, Pipelines as Code would not report a pipeline run creation error when Tekton Pipelines had issues creating the `PipelineRun` resource. For example, a non-existing task in a pipeline run would show no status. With this update, PAC shows the proper error message coming from `tekton/pipeline` along with the task that is missing.
```

Comment:
```suggestion
* Before this update, Pipelines as Code would not report a pipeline run creation error when Tekton Pipelines had issues creating the `PipelineRun` resource. For example, a non-existing task in a pipeline run would show no status. With this update, PAC shows the proper error message coming from `tekton/pipeline` along with the task that is missing.
```
The `tekton/pipeline` literal needs a modifier. 

Check here for more information about modifiers for objects and literal values: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references